### ISSUE TITLE
Excessive time to analyze submissions for Java

### ISSUE THREAD

**Issue by jdearly** [exercism/exercism#6325]
Hello! Code analysis seems to be stuck for any exercise that has been submitted on the Java tack in the last few days. Each submission passes all the tests, but remains "Processing" seemingly indefinitely. 

![processing](https://user-images.githubusercontent.com/22900979/162971933-630f720a-06de-49c6-8544-03e7b4bcdba3.png)

**Comment by tdjordjevski** [comment]
It seems the issue maybe site-wide. I'm seeing the same problem with F#.
![image](https://user-images.githubusercontent.com/49134026/163093176-be88aecd-5fd8-4b67-b6ac-e7ce9e63f75b.png)

**Comment by SleeplessByte** [comment]
They sometimes get stuck, indeed.

cc @ErikSchierboom @iHiD

**Comment by philg-dev** [comment]
I've got this same problem for all my exercises as well.

Mine were all in Java:
- Hello World
- Word Search
- Queen Attack

All are stuck at the "Analysis" part, despite having passed all tests, etc.

I did submit my solutions via CLI, if that matters.

**Comment by iHiD** [comment]
@ErikSchierboom Could you did into this pls?

**Comment by ErikSchierboom** [comment]
@iHiD The first step in the analysis: all above solutions have their latest iteration's `analysis_status` property set to `not_queued`.

**Comment by ErikSchierboom** [comment]
> Is there a common representation_status property too?

There is. They're all `exceptioned`

**Comment by ErikSchierboom** [comment]
`has_analyzer` is set to false

**Comment by ErikSchierboom** [comment]
I've opened a PR to fix this: https://github.com/exercism/website/pull/2621

TL;DR The website _thinks_ analysis is being done in the background, but that is not true because the track doesn't have an (activated) analyzer.

**Comment by MrYarrow** [comment]
Same happening here for Java HelloWorld

**Comment by ErikSchierboom** [comment]
@MrYarrow If you go to your hello-world solution now, it should be fixed. Could you check?

**Comment by philg-dev** [comment]
The problem is still the same on my end. (For all three of the exercises I did and mentioned before)

I tried CTRL+F5 (no cache refresh) and manually clearing the site's cache & cookies, but it didn't help either.

**Comment by MrYarrow** [comment]
> @MrYarrow If you go to your hello-world solution now, it should be fixed. Could you check?

No, still processing on all tracks...
Python works though ;-)

**Comment by ErikSchierboom** [comment]
We've identified the problem and will work on fixing it.

**Comment by ErikSchierboom** [comment]
@MrYarrow Could you check again?

**Comment by philg-dev** [comment]
For me the exercises appear correctly as "passed" now. Thanks a lot 👍

**Comment by ErikSchierboom** [comment]
Yay! Thanks for reporting this issue.

**Comment by MrYarrow** [comment]
> @MrYarrow Could you check again?

Yes, for me it works now, too! :)
